studiopress/genesis-custom-blocks
# Add 'Editor Preview' and 'Front-end Preview'

#### Changes
* Adds 'Editor Preview' and 'Front-end Preview':

![some-previews](https://user-images.githubusercontent.com/4063887/107828755-1d848480-6d4f-11eb-934f-90f4aadcb05e.gif)

#### Todo
- [x] Make the edits in the 'Editor Preview' persist across page loads

#### Testing instructions
1. `/wp-admin` > Custom Blocks > Add New
2. Add a few fields
3. Click 'Editor Preview' 
4. Expected: Those fields display as they will in the editor
5. Add a PHP template for the block
6. Click 'Front-end Preview'
7. Expected: The template displays
8. Add a `.css` file for the block
9. Expected: the styling from it appears in the editor.
